.github/workflows/build-project.yaml

```diff
@@ -90,7 +94,7 @@ jobs:
           print "pluginName=${product_name}" >> $GITHUB_OUTPUT
           print "pluginVersion=${product_version}" >> $GITHUB_OUTPUT
 
-      - uses: actions/cache@v3
+      - uses: actions/cache/restore@v3
```

Comment:
While this will probably not have any negative side-effect on its own, the change on `obs-studio` was connected to explicit calls to _save_ the cache in other parts of the workflow.

I'd rather not introduce this change here as the PR is unrelated to those changes.

Reply:
Fixed